resources/views/page/overview.blade.php

```diff
@@ -9,7 +9,9 @@
             <h1 class="font-bold text-4xl mb-5">{{ $page->content_heading }}</h1>
         @endif
         @includeIf('pages.'.$page->identifier)
-        @widget('content', 'pages', ($page->identifier == 'home' ? 'cms' : $page->identifier).'_index_index')
+        <div class="hidden lg:block">
+            @widget('content', 'pages', ($page->identifier == 'home' ? 'cms' : $page->identifier).'_index_index')
+        </div>
```

Comment:
This would erase the blocks and the magento content, such as the hot sellers slider. The blocks are hidden on mobile due to their lack of responsiveness.